internal/testgen/gen_empty.go

```diff
@@ -163,30 +212,43 @@ import (
 )
 
 func {{ .CheckerName.AsTestName }}(t *testing.T) {
-	var elems []string
+	var (
+		elems []any
+		str   string
+		b     []byte
+	)
 	{{ range $ti, $test := $.Tests }}
 		// {{ $test.Name }}.
 		{
 			// Invalid.
 			{{- range $ai, $assrn := $test.InvalidAssertions }}
+				{{- range $vi, $var := $test.Vars }}
+					{{ NewAssertionExpander.Expand $assrn "assert" "t" (arr $var) }}
+				{{- end }}
+			{{- end }}
+			{{- range $ai, $assrn := $test.Special }}
 				{{ NewAssertionExpander.Expand $assrn "assert" "t" nil }}
 			{{- end }}
```

Comment:
What I suggested here

https://github.com/Antonboom/testifylint/pull/129#discussion_r1964308416


Might be also be interesting here, so the comment would be in the generated file

Reply:
add comment, that `Zero` covers strings only